Capstone/Assets/Game/Scripts/StateMachine/PlayerStates/PlayerFSMState_BLOCK.cs

```diff
@@ -38,6 +40,29 @@ public class PlayerFSMState_BLOCK : PlayerFSMState
 
         public override void OnEnter()
         {
+            camera = GameObject.FindGameObjectWithTag("MainCamera");
+
+            if (_input.move == Vector2.zero)
+            {
+                _targetRotation = _player.transform.eulerAngles.y;
+
+                inputDirection = Quaternion.Euler(0.0f, _targetRotation, 0.0f) * Vector3.forward;
+            }
+            else
+            {
+                Vector3 inputMove = new Vector3(_input.move.x, 0.0f, _input.move.y).normalized;
+
+                // rotate to face input direction relative to camera position
+                _targetRotation = Mathf.Atan2(inputMove.x, inputMove.z) * Mathf.Rad2Deg + camera.transform.eulerAngles.y;
+
+                _player.transform.rotation = Quaternion.Euler(0.0f, _targetRotation, 0.0f);
```

Comment:
Can you save the `Quaternion.Euler(0.0f, _targetRotation, 0.0f);` in a variable since you're using it like the next line?